reactive/kubernetes_master_worker_base.py

```diff
@@ -13,6 +16,47 @@
 from charmhelpers.core import hookenv
 from charmhelpers.core.host import is_container
 from charmhelpers.core.sysctl import create as create_sysctl
+from charms.layer.kubernetes_common import arch
+import os
+from subprocess import check_call
+
+
+@hook('upgrade-charm')
+def upgrade_charm():
+    clear_flag('kubernetes.cni-plugins.installed')
+
+
+@when_not('kubernetes.cni-plugins.installed')
```

Comment:
is it not necessary to use decorator

```python
@when('endpoint.container-runtime.available')
```

I see it was removed from CK-worker

Reply:
I couldn't find any reason to keep it. The handler doesn't depend on the container-runtime relation in any way. We don't need containerd to be running before we make /opt/cni/bin and put stuff in there. I also didn't find any handlers waiting for `kubernetes-worker.cni-plugins.installed`, so they wouldn't be relying on that to imply `endpoint.container-runtime.available` is also true or anything like that.

Reply:
perfect.  thanks for the explanation.  
LGTM :+1: